CHANGES.rst

```diff
@@ -22,7 +22,8 @@ Bug Fixes
 
 - ``ccd_process`` now copies the meta of the input when subtracting the
   master bias. [#404]
-
```

Comment:
Should the number in brackets in CHANGES refer to the Issue #421  or the PR #422 ?